### PR TITLE
ESDT Safe Cleanup

### PR DESCRIPTION
**Pull request by andreiblt1304**

### CODE REVIEW

**Review by mihaicalinluca — APPROVED** [review]

**Review by alyn509 — APPROVED** [review]

File: esdt-safe/src/to_sovereign/create_tx.rs
Comment (axenteoctavian) [inline esdt-safe/src/to_sovereign/create_tx.rs:1]: - the first 6 endpoints should be deleted
- put `deposit` endpoint as first function in this file

File: esdt-safe/src/from_sovereign/token_mapping.rs [hunk 1]
```diff
@@ -12,54 +11,6 @@ pub struct EsdtTokenInfo<M: ManagedTypeApi> {
 pub trait TokenMappingModule:
     multiversx_sc_modules::default_issue_callbacks::DefaultIssueCallbacksModule
 {
-    #[payable("EGLD")]
-    #[endpoint(registerToken)]
-    fn register_token(
```
Comment (axenteoctavian) [inline esdt-safe/src/from_sovereign/token_mapping.rs:17]: Please revert this change.
In `esdt-safe` contract we still use `register_token`. Only in `enshrine-esdt` we don't use it anymore.

File: esdt-safe/src/esdt_safe_proxy.rs
Comment (axenteoctavian) [inline esdt-safe/src/esdt_safe_proxy.rs:1]: I'm not sure how this proxy file works but I have a question. 
None of the contracts have calls to esdt-safe, so do we need this proxy file?

**Review by axenteoctavian — CHANGES_REQUESTED** [review]
Please resolve the comments

File: esdt-safe/src/to_sovereign/create_tx.rs
Comment (andreiblt1304) [reply esdt-safe/src/to_sovereign/create_tx.rs:1]: done

File: esdt-safe/src/to_sovereign/events.rs
[the remark this answers is not in the document]
Comment (andreiblt1304) [reply esdt-safe/src/to_sovereign/events.rs:1]: deleted

File: esdt-safe/src/from_sovereign/token_mapping.rs (hunk 1 above)
Comment (andreiblt1304) [reply esdt-safe/src/from_sovereign/token_mapping.rs]: reverted

File: esdt-safe/src/esdt_safe_proxy.rs
Comment (andreiblt1304) [reply esdt-safe/src/esdt_safe_proxy.rs:1]: We have a script to generate proxies for all the contracts, It is far easier to have all the proxies generated inside the src folder than to create them manually and then drag and drop to the correct path. Maybe in the future we will use that proxy.

**Review by axenteoctavian — APPROVED** [review]
